fdbserver/RestoreLoader.actor.cpp

```diff
@@ -196,6 +181,29 @@ ACTOR Future<Void> handleLoadFileRequest(RestoreLoadFileRequest req, Reference<R
 	ASSERT(self->processedFileParams.find(req.param) != self->processedFileParams.end());
 	wait(self->processedFileParams[req.param]); // wait on the processing of the req.param.
 
+	// TODO: Send sampled mutations back to master
+	req.reply.send(RestoreCommonReply(self->id()));
+	return Void();
+}
+
+ACTOR Future<Void> handleSendMutationsRequest(RestoreSendMutationsToAppliersRequest req,
+                                              Reference<RestoreLoaderData> self) {
+	if (self->rangeToApplier.empty()) {
+		self->rangeToApplier = req.rangeToApplier;
+	} else {
+		ASSERT(self->rangeToApplier == req.rangeToApplier);
+	}
+
+	// Send mutations from log files first to ensure log mutation at the same version is before the range kv
+	state std::map<LoadingParam, VersionedMutationsMap>::iterator item = self->kvOpsPerLP.begin();
+	for (; item != self->kvOpsPerLP.end(); item++) {
+		if (item->first.isRangeFile == req.useRangeFile) {
+			// Send the parsed mutation to applier who will apply the mutation to DB
+			wait(sendMutationsToApplier(self, &item->second, item->first.isRangeFile, item->first.prevVersion,
+			                            item->first.endVersion, item->first.fileIndex));
```

Comment:
This API can be simplified as `sendMutationsToApplier(self, item->first, item->second)`.

Reply:
I prefer not to use the `LoadingParam` in the param list right now because the fields in `LoadingParam` is not finally decided yet. In addition, unless we will use all LoadingParam's fields, passing the only-necessary variables makes it easier to interpret what the function needs as input.